hash/fmtowns_cd.xml

```diff
@@ -8586,6 +8680,24 @@ User/save disks that can be created from the game itself are not included.
 		</part>
 	</software>
 
+	<!-- Missing a floppy disk, the game doesn't work without it -->
+	<software name="suikoden">
```

Comment:
If the game doesn't work, it should be marked as ```supported="no"``` in the ```<software>``` tag.

Reply:
Done.